src/main/java/org/craftercms/studio/api/v2/utils/GitRepositoryHelper.java

```diff
@@ -540,6 +546,7 @@ public boolean createPublishedRepository(String siteId, String sandboxBranch) {
             Repository publishedRepo = publishedGit.getRepository();
             optimizeRepository(publishedRepo);
             checkoutSandboxBranch(siteId, publishedRepo, sandboxBranch);
+            removePublishBlackList(publishedRepo);
```

Comment:
@phuongnq if the operation is done in the sandbox branch we might need to merge it into the `live` branch

Reply:
Hi @jdrossl 
Thanks for the comment.
AFAIK, the `live` branch will be created after we created this published repository(?)
If this is the case, the new commit will be automatically merged to the newly created branch switch. Do we have any case that which the `live` branch is created beforehand?

Reply:
We have to test it with initial publish, regular publish & the new publish all once that branch is merged

Reply:
I will send a video for detail